src/modules/ie-objects/elasticsearch/elasticsearch.consts.ts

```diff
@@ -202,3 +202,11 @@ export const NEEDS_AGG_SUFFIX: { [prop in IeObjectsSearchFilterField]?: string }
 	[IeObjectsSearchFilterField.GENRE]: 'keyword',
 	[IeObjectsSearchFilterField.TYPE]: 'keyword',
 };
+
+/*
+	This prefix is added before and after every digit before jsep parses it.
+	This has to be a string that's not expected to occur in any ES object's properties.
+	It should also be made of characters that aren't recognised by jsep.
+	If the searchterm is a string starting with a digit and we don't do this, jsep will throw an error saying a variable cannot start with a digit.
+*/
+export const ES_DIGIT_PREFIX = '_ç_ç_ç_ç_ç_';
```

Comment:
This has nothing to do with elasticsearch. It is only used before and after the parsing with jsep
```suggestion
export const JSEP_DIGIT_PREFIX = '_ç_ç_ç_ç_ç_';
```